### PR TITLE
altered header

### PR DESCRIPTION
**Pull request by shashanktester**
Please review

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <h1>
     Welcome to {{ title }} change done by shashanktester!
   </h1>
-  <h2>New Header</h2>
+  <h2>New Header For Angular</h2>
```
Comment (ShashankJoshi007) [inline src/app/app.component.html]: Why it is altered

Comment (shashanktester) [reply src/app/app.component.html]: It is altered by my wish

**Comment by ShashankJoshi007** [comment]
Ok done

**Comment by ShashankJoshi007** [comment]
Ok

**Pull request merged**